usama/sprint1/cdk.json

```diff
@@ -0,0 +1,18 @@
+{
+  "app": "python3 app.py",
```

Comment:
- I think this file should be ignored, **your .gitignore file is not working well**

Reply:
Updated.